### PR TITLE
feat: Add Release Health for JS

### PR DESCRIPTION
**Pull request by HazAT**
Reworked content so it also works for JS.

~Also added `autoSessionTracking: true` to the wizard.~

### CODE REVIEW

**Review by PeloWriter — APPROVED** [review]

File: src/includes/configuration/auto-session-tracking/android.mdx [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+Or, if you are manually instrumenting Sentry:
+
+```java
+import io.sentry.android.core.SentryAndroid;
+
+SentryAndroid.init(
+  options -> {
+    options.setSessionTrackingIntervalMillis(10000);
+  });
```
Comment (marandaneto) [inline src/includes/configuration/auto-session-tracking/android.mdx:17]: the text says:

> For example, to configure it to be 60 seconds

but the code snippet os 10000 = 10s 

either fix the code snippets or change the text

File: src/includes/configuration/auto-session-tracking/flutter.mdx [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+```dart
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+Future<void> main() async {
+  await SentryFlutter.init(
+    (options) => options.autoSessionTrackingIntervalMillis = 10000, // it's 30000 milliseconds (30s) by default
+    appRunner: () => runApp(MyApp()),
+  );
+}
```
Comment (marandaneto) [inline src/includes/configuration/auto-session-tracking/flutter.mdx:12]: same as https://github.com/getsentry/sentry-docs/pull/2743/files#r536377858

I'd also remove the comment `// it's 30000 milliseconds (30s) by default`
as the text already mentions that

File: src/platforms/common/configuration/releases.mdx [hunk 3]
```diff
@@ -43,21 +43,13 @@ This tags each event with the release value. We recommend that you tell Sentry a
 
 After configuring your SDK, you can install a repository integration or manually supply Sentry with your own commit metadata. Read our documentation about [Releases](/product/releases/) for further information about integrations, associating commits, and telling Sentry when deploying releases.
 
-<PlatformSection supported={["apple"]}>
+<PlatformSection supported={["apple", "android", "javascript"]}>
```
Comment (marandaneto) [inline src/platforms/common/configuration/releases.mdx:46]: should we add `flutter`? react-native also has support for release health

Comment (HazAT) [reply src/platforms/common/configuration/releases.mdx:46]: Good catch!

File: src/includes/configuration/auto-session-tracking/android.mdx (hunk 1 above)
Comment (HazAT) [reply src/includes/configuration/auto-session-tracking/android.mdx:17]: Fixed

File: src/includes/configuration/auto-session-tracking/flutter.mdx (hunk 2 above)
Comment (HazAT) [reply src/includes/configuration/auto-session-tracking/flutter.mdx:12]: Fixed

Comment (marandaneto) [reply src/includes/configuration/auto-session-tracking/flutter.mdx:12]: guess you forgot to commit this one

File: src/includes/configuration/auto-session-tracking/flutter.mdx [hunk 4]
```diff
@@ -1,3 +1,19 @@
+By default, the session is terminated once the application is in the background for more than 30 seconds. You can change the time out with the option named `sessionTrackingIntervalMillis`. It takes the amount in milliseconds. For example, to configure it to be 60 seconds:
+
+```dart
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+Future<void> main() async {
+  await SentryFlutter.init(
+    (options) => options.autoSessionTrackingIntervalMillis = 10000, // it's 30000 milliseconds (30s) by default
```
Comment (marandaneto) [inline src/includes/configuration/auto-session-tracking/flutter.mdx:9]:
```suggestion
    (options) => options.autoSessionTrackingIntervalMillis = 60000
```

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**